src/material-experimental/package.json

```diff
@@ -23,7 +23,12 @@
     }
   },
   "peerDependencies": {
+    "@angular/animations": "0.0.0-NG",
+    "@angular/cdk": "0.0.0-PLACEHOLDER",
     "@angular/core": "0.0.0-NG",
+    "@angular/common": "0.0.0-NG",
+    "@angular/forms": "0.0.0-NG",
+    "@angular/platform-browser": "0.0.0-NG",
```

Comment:
Technically these are already listed as `peerDependencies` on `@angular/material` which was already listed, but it might be better to list them explicitly.